content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -557,3 +547,15 @@ replenishment costs more effectively.
 
 .. image:: reordering_rules/counterexample.png
    :alt: Example where the visibility days does not trigger.
+
+Horizon days
+============
+
+*Horizon days* determine how many days ahead Odoo checks if the forecasted quantity will drop below
+reordering rule's minimum. The feature is meant to help users plan replenishment in advance, by
+increasing the :ref:`forecasted date <inventory/warehouses_storage/forecasted-date>`.
+
+.. math:: Forecasted~date = Current~date + Vendor~Lead~Time + Horizon~Days
```

Comment:
Small changes here:
- add the `:class:` attribute with the `overflow-scroll` class on long math equations for better mobile formatting
- use the [`\text{ }`](https://www.docutils.org/docs/ref/rst/mathematics.html#text) command to exclude text from formula formatting :)
```suggestion
.. math::
   :class: overflow-scroll

   \text{Forecasted date} = \text{Current date} + \text{Vendor Lead Time} + \text{Horizon Days}

```